### PR TITLE
Merged from openstack-ansible: Implements host_routes option for neutron subnet

### PR DESCRIPTION
**Pull request by toddnni**
I merged cdwertmann commit to quantum_subnet. 

I understand that there is a major rework going on with OpenStack modules, but this is a minor fix.

Conflicts:
    cloud/openstack/quantum_subnet.py

### CODE REVIEW

**Comment by gregdek** [comment]
@emonty your opinion here?

**Comment by gregdek** [comment]
@emonty pingy ping

**Comment by emonty** [comment]
Lgtm ... Let's do it
On Jul 23, 2015 1:57 PM, Greg DeKoenigsberg notifications@github.com wrote:@emonty pingy ping

File: cloud/openstack/quantum_subnet.py [hunk 1]
```diff
@@ -114,6 +114,11 @@
         - From the subnet pool the last IP that should be assigned to the virtual machines
      required: false
      default: None
+   host_routes:
+     description:
+        - Host routes for this subnet, list of dictionaries, e.g. [{destination: 0.0.0.0/0, nexthop: 123.456.78.9}, {destination: 192.168.0.0/24, nexthop: 192.168.0.1}]
+     required: false
+     default: None
```
Comment (bcoca) [inline cloud/openstack/quantum_subnet.py:121]: needs version_added

**Comment by gregdek** [comment]
@toddnni Thanks for submitting this pull request. Unfortunately, it appears that the underlying code has changed since your submission, and this PR now needs to be rebased. 

If you believe that this PR still applies, please rebase and we will put it into our community review process as soon as possible. 

Once you have rebased this PR, please add a comment with the text "ready_for_review" and we will continue with the review process.  Thanks!

**Comment by gregdek** [comment]
@toddnni A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by toddnni** [comment]
@gregdek, I'm sorry but I haven't been working with the modules lately. I took a quick glance to master branch and it seems that the os_subnet.py already covers 'host_routes'. This issue can be closed.

**Pull request closed without merging**